src/yb/master/master_main.cc

```diff
@@ -69,6 +71,16 @@ static int MasterMain(int argc, char** argv) {
   // Reset some default values before parsing gflags.
   FLAGS_rpc_bind_addresses = strings::Substitute("0.0.0.0:$0", kMasterDefaultPort);
   FLAGS_webserver_port = kMasterDefaultWebPort;
+
+
+  auto host_name = GetHostname();
+  if (host_name.ok()) {
+    FLAGS_metric_node_name = (*host_name) + ":" + std::to_string(FLAGS_webserver_port);
+  }
+  else {
+      LOG(INFO) << "Failed to get current host name, keeping default metric_node_name: " << host_name.status();
```

Comment:
This line is longer than 100 chars, which is the limit by convention in this code base.